R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -95,10 +97,53 @@ TableHandle <- R6Class("TableHandle",
     to_data_frame = function() {
       arrow_tbl <- self$to_arrow_table()
       return(as.data.frame(as.data.frame(arrow_tbl))) # TODO: for some reason as.data.frame on arrow table returns a tibble, not a data frame
-    }
+    },
+    internal_table_handle = NULL
   ),
   private = list(
-    internal_table_handle = NULL,
     is_static_field = NULL
   )
 )
+
+# EXISTING GENERICS THAT WE CAN SUPPORT
```

Comment:
Because these are generics, they already have docstrings that tell you what they should do. There may be a `?as.data.frame.TableHandle` to get the docstrings specifically for this implemented generic, but I'm not sure.